### DISCUSSION TITLE
[ERROR] Dependency spritesheet-js must be installed to proceed

### DISCUSSION THREAD

**Discussion by uploadx10** [Kas-tle/java2bedrock.sh#19] [Converter Help] [answered]
I have installed spritesheet-js, the system can't find it.

[Amory.zip](https://github.com/Kas-tle/java2bedrock.sh/files/8792348/Amory.zip)

**Comment by uploadx10** [comment]
https://i.imgur.com/kRvAOly.png

**Comment by uploadx10** [comment]
i can fix it

**Comment by EbaaCode** [comment]
We need more information to help you.

What steps have you taken so far to try resolve this problem?
What operating system are you using?

If you're using WSL you may have to do this to make it work:
> If you install spritesheet-js globally, but the script fails due to missing the dependency, ensure that the folder containing your global node modules is added to your path. If, after installing, `which spritesheet-js` returns no output, you may add global node modules to your path by:
> ```sh
>  echo "export PATH=$PATH:$(npm config --global get prefix)/bin" >> ~/.bashrc && source ~/.bashrc
> ```

**Comment by Kas-tle** [comment, marked as answer]
@uploadx10 do note if you are using MacOS newer versions use zsh (you can check with `echo $SHELL`). If that's the case, you would need to place the export in `~/.zshrc` and soucre from there.
